progress/meter.go

```diff
@@ -48,7 +52,7 @@ func NewProgressMeter(estFiles int, estBytes int64, dryRun bool, logPath string)
 		finished:       make(chan interface{}),
 		estimatedFiles: int32(estFiles),
 		estimatedBytes: estBytes,
-		dryRun:         dryRun,
+		DryRun:         dryRun,
```

Comment:
Going to un-export this, move `Start()` to this constructor, and then add a `MeterOption` func to set `dryRun`.